cmd/porter/mixins.go

```diff
@@ -26,25 +26,23 @@ func buildMixinsCommand(p *porter.Porter) *cobra.Command {
 }
 
 func buildMixinsListCommand(p *porter.Porter) *cobra.Command {
-	opts := struct {
-		rawFormat string
-		format    printer.Format
-	}{}
+	opts := porter.MixinListOptions{}
+
 	cmd := &cobra.Command{
 		Use:   "list",
 		Short: "List installed mixins",
 		PreRunE: func(cmd *cobra.Command, args []string) error {
 			var err error
-			opts.format, err = printer.ParseFormat(opts.rawFormat)
+			opts.Format, err = printer.ParseFormat(opts.RawFormat)
 			return err
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return p.PrintMixins(printer.PrintOptions{Format: opts.format})
+			return p.PrintMixins(printer.PrintOptions{Format: opts.Format})
```

Comment:
The intent was that instead of matching the opts to the cli command (MixinList), was to match the opts to the porter command being executed (PrintMixins). So this would be `p.PrintMixins(opts)`, where opts would be of type `PrintMixinsOptions`.

Reply:
Ah, thank you.  Updated.